pkgs/top-level/all-packages.nix

```diff
@@ -13198,13 +13198,10 @@ with pkgs;
 
   qgo = libsForQt5.callPackage ../games/qgo { };
 
-  anki = callPackage ../games/anki {
-    protobuf = protobuf_31;
-  };
-  anki-utils = callPackage ../games/anki/addons/anki-utils.nix { };
-  ankiAddons = recurseIntoAttrs (callPackage ../games/anki/addons { });
-  anki-bin = callPackage ../games/anki/bin.nix { };
-  anki-sync-server = callPackage ../games/anki/sync-server.nix { };
+  anki-utils = callPackage ../by-name/an/anki/addons/anki-utils.nix { };
+  ankiAddons = recurseIntoAttrs (callPackage ../by-name/an/anki/addons { });
+  anki-bin = callPackage ../by-name/an/anki/bin.nix { };
+  anki-sync-server = callPackage ../by-name/an/anki/sync-server.nix { };
```

Comment:
It looks to me like `anki-bin` and `anki-sync-server` have no paths pointing at any of the other files - that means they can go into their own `by-name/an/anki-{bin,sync-server}` folders.